avsomr/avsomrreader.h

```diff
@@ -0,0 +1,51 @@
+#ifndef AVS_AVSOMRREADER_H
+#define AVS_AVSOMRREADER_H
+
+#include <memory>
+#include <QImage>
```

Comment:
this too

Reply:
I did not find QImage in the all.h, add QImage in the all.h?

Reply:
Yes, I think so